doc/source/user_guide/make_container_image.rst

```diff
@@ -0,0 +1,99 @@
+.. _ref_make_container:
+
+Create Fluent Docker container image
+====================================
+
+.. warning:: You need a valid Ansys license and an Ansys account to
+   follow the steps in this section.
+
+The following procedure for creating a Fluent Docker container image uses a
+Linux machine to copy the needed files from the Fluent installation directory.
+
+
+Prerequisites
+-------------
+
+* A Linux machine with `Docker <https://www.docker.com>`_ installed.
+
+* A valid Ansys license and account. Your Ansys reseller should have provided you with one.
```

Comment:
"with one" refers to what? There are two things mentioned in the previous sentence.

Reply:
We need license only as the Ansys is pre-installed. Done. Thank you.